en/docs/learn/extension-points-for-oauth.md

```diff
@@ -557,7 +557,30 @@ Implementations supporting both JSON and JWT response formats are available. By
 </tbody>
 </table>
 
-  
+### Adding a custom GrantTypeValidatorImplClass
+
+<table>
+<tbody>
+<tr class="odd">
+<td>Usage</td>
+<td><p>This can be used to add custom <code>GrantTypeValidatorImplClasses</code> to the <code>refresh_token</code> grant type<br /></td>
```

Comment:
```suggestion
<td><p>This can be used to add a custom <code>GrantTypeValidatorImplClass</code> to the <code>refresh_token</code> grant type<br /></td>
```